tests/unit/test_binning_model.py

```diff
@@ -0,0 +1,42 @@
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.models.binning_model import BinningModelObject
+
+
+@responses.activate
+def test_binning_model():
+
+    project_config = {
+        "name": "Project 1",
+        "description": "Mastering Project",
+        "type": "DEDUP",
+        "unifiedDatasetName": "Project 1 - Unified Dataset",
+        "externalId": "Project1",
+        "resourceId": "1",
+    }
+
+    binning_model_json = {
+        "id": ["d8b7351d-24ce-49aa-8655-5b5809ab6bb8"],
+        "isActive": ["true"],
+        "clauseId": ["2e6c5f1b-ed49-40ab-8cbb-350aded25070"],
+        "similarityFunction": ["COSINE"],
+        "tokenizer": ["DEFAULT"],
+        "fieldName": ["surname"],
+        "threshold": ["0.75"],
+    }
+
+    unify = Client(UsernamePasswordAuth("username", "password"))
+
+    project_id = "1"
+
+    project_url = f"http://localhost:9100/api/versioned/v1/projects/{project_id}"
+    binning_model_url = f"http://localhost:9100/api/versioned/v1/projects/{project_id}/binningModel/records"
+
+    responses.add(responses.GET, project_url, json=project_config)
+    responses.add(responses.GET, binning_model_url, json=binning_model_json)
```

Comment:
This is actually not quite right; see tests/unit/test_dataset_records.py for example.
Because `records()` isn't expecting JSON (it's expecting newline-separated serialized JSON), we can't provide a `json=` - we need to provide a `body=` with newline-separated, serialized JSON.